fixtures/loader.go

```diff
@@ -293,8 +293,7 @@ func (f *Loader) loadTable(ctx *loadContext, t string, rows table) error {
 			// add to references
 			ctx.refsDefinition[name] = row
 			if f.debug {
-				rowJson, _ := json.Marshal(values)
-				fmt.Printf("Populating ref %s as %s from row definition\n", name, string(rowJson))
+				fmt.Printf("Populating ref %s as %s from row definition\n", name, row)
```

Comment:
Why do you consider that output is wrong? I guess it is intentionally displaying the row content.

Reply:
Because a few lines below there is a line that prints the same debug output. Here row definition should be printed, which is `row`. And below will be printed row content, which is `values`.

Reply:
OK, right. You have better to convert it into JSON before printing.

Reply:
Done